src/status_im2/contexts/shell/activity_center/events.cljs

```diff
@@ -102,7 +102,10 @@
 (rf/defn reconcile-seen-state
   {:events [:activity-center/reconcile-seen-state]}
   [{:keys [db]} seen?]
-  {:db (assoc-in db [:activity-center :seen?] seen?)})
+  (cond-> {:db (assoc-in db [:activity-center :seen?] seen?)}
+
+    (= (:view-id db) :activity-center)
```

Comment:
I understand using `view-id` can be necessary but it is so prone to introducing simple bugs. I think it's best to cover this with the use of a test 👍